[pytorch] Temporarily disable test_numerical_consistency_per_tensor

jjlilley opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#30600 [pytorch] Temporarily disable test_numerical_consistency_per_tensor**

test_numerical_consistency_per_tensor in test_fake_quant is failing on Windows.

Differential Revision: [D18760287](https://our.internmc.facebook.com/intern/diff/D18760287/)

Commit:
[pytorch] Temporarily disable test_numerical_consistency_per_tensor

test_numerical_consistency_per_tensor in test_fake_quant is failing on Windows.

Differential Revision: [D18760287](https://our.internmc.facebook.com/intern/diff/D18760287/)

[ghstack-poisoned]

suo reviewed (approved):
Can you file an issue for this test and link it in the disable message please.

Commit:
Update on "[pytorch] Temporarily disable test_numerical_consistency_per_tensor"


test_numerical_consistency_per_tensor in test_fake_quant is failing on Windows.

Differential Revision: [D18760287](https://our.internmc.facebook.com/intern/diff/D18760287/)

[ghstack-poisoned]

jjlilley commented:
Sure, I just filed and issue and added a comment with a link
